### PR TITLE
Update of example notebooks

### PR DESCRIPTION
**Pull request by avvliet**
I've checked the current notebook examples on typos and outdated code. There are still some deprecation warnings and I'm sure that there are more things that can be improved, but, for now, this is already a relevant improvement I would say.

### CODE REVIEW

File: doc/pages/example_notebooks/density/densitys_easy_example.ipynb [hunk 1]
```diff
@@ -184,16 +199,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## customize a density with different density models\n",
+    "## Customize a density with different density models.\n",
     "To customize a density use the DensityList.\n",
-    "In a superposition of globel models of density distribution of the Milkyway keep care of normalisation. Therfore you can just add components by deactivating the other  "
+    "In a superposition of global models of the density distribution of the Milky Way take care of the normalisation. For this purpose you can just add components after deactivating the unwanted parts.  "
```
Comment (lukasmerten) [inline doc/pages/example_notebooks/density/densitys_easy_example.ipynb:204]: This sentence does not have a subject. I am also not sure what we want to say here. 
Does it mean that the user has to calculate the normalization on their own?

File: doc/pages/example_notebooks/density/densitys_model_example.ipynb [hunk 2]
```diff
@@ -4,11 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# example for the implemented massdistributions of the Milky Way\n",
+    "# Example for the implemented mass distributions of the Milky Way\n",
```
Comment (lukasmerten) [inline doc/pages/example_notebooks/density/densitys_model_example.ipynb:7]: Maybe we should better speak of "gas density distributions", making clear that we are not referring to, e.g., the matter bound in stars.

**Review by lukasmerten — COMMENTED** [review]
The updates look good. However, at the moment I have not run all notebooks again. I suggest to merge after my small comments are included. Afterwards we can go through the notebooks again and take a look a deprecation warnings etc. 

Make sure to not override the changes made in the diffusion examples, which I just merged.

File: doc/pages/example_notebooks/density/densitys_easy_example.ipynb (hunk 1 above)
Comment (avvliet) [reply doc/pages/example_notebooks/density/densitys_easy_example.ipynb]: I've changed this sentence in the last commit. I think it means that the script given there shows how to take into account the normalization properly for a superposition of global models.

File: doc/pages/example_notebooks/density/densitys_model_example.ipynb (hunk 2 above)
Comment (avvliet) [reply doc/pages/example_notebooks/density/densitys_model_example.ipynb]: I agree, I've changed it.

**Review by lukasmerten — APPROVED** [review]

**Pull request merged**